lib/rubocop/cop/style/fetch_env_var.rb

```diff
@@ -39,7 +39,7 @@ module Style
       #   !ENV['X']
       #   ENV['X'].some_method # (e.g. `.nil?`)
       #
-      class FetchEnvVar < Base
+      class FetchEnvVar < Base # rubocop:disable Metrics/ClassLength
```

Comment:
Can you rebase with the latest master branch to remove this disable comment?

```suggestion
      class FetchEnvVar < Base
```